base/applications/mspaint/lang/ro-RO.rc

```diff
@@ -17,8 +17,8 @@ BEGIN
     BEGIN
         MENUITEM "&Nou\tCtrl+N", IDM_FILENEW
         MENUITEM "&Deschidere...\tCtrl+O", IDM_FILEOPEN
-        MENUITEM "&Păstrează\tCtrl+S", IDM_FILESAVE
-        MENUITEM "Păst&rare în...", IDM_FILESAVEAS
+        MENUITEM "S&alvează\tCtrl+S", IDM_FILESAVE
```

Comment:
Accelerator &a collides with
 MENUITEM "Decor de birou (în c&arou)", IDM_FILEASWALLPAPERPLANE
in the same menu.